### PR TITLE
Dialog

### PR DESCRIPTION
**Pull request by matbyl**

### CODE REVIEW

File: app/src/Stories/Text.elm [hunk 1]
```diff
@@ -55,12 +54,12 @@ view =
...
-                [ Typography.styledText TextStyle.heading1 "Headline 1"
-                , Typography.styledText TextStyle.heading2 "Headline 2"
-                , Typography.styledParagraph TextStyle.body """
+                [ Text.view TextStyle.heading1 "Headline 1"
+                , Text.view TextStyle.heading2 "Headline 2"
+                , Text.customParagraph TextStyle.body """
                     Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec non magna lorem. Integer ultrices, odio sit amet tincidunt semper, augue odio elementum tellus, sollicitudin varius nisi magna vitae nunc. Nulla ornare tincidunt ultrices. Nulla pulvinar ligula volutpat tellus commodo, ut lobortis nisl aliquet. Suspendisse scelerisque pharetra risus, ac ultricies ligula sodales ac. Pellentesque tempor, ligula sit amet lacinia egestas, velit nunc bibendum erat, vel ornare mi sem non lacus. Pellentesque congue sagittis sem laoreet vulputate. Aliquam erat volutpat. 
                     """
-                , Typography.styledParagraph TextStyle.body """
+                , Text.customParagraph TextStyle.body """
```
Comment (balazs-lengyel) [inline app/src/Stories/Text.elm:62]: change textstyle to != body

File: src/Ui/Input.elm [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+import Ui.Button
+import Ui.Color exposing (toCssColor)
+import Ui.Icon as Icon
+import Ui.Palette
+
+
+inputTextBaseStyle : Attribute msg
+inputTextBaseStyle =
+    css
+        [ Css.padding4 (Css.px 4) (Css.px 5) (Css.px 4) (Css.px 5)
```
Comment (balazs-lengyel) [inline src/Ui/Input.elm:20]: `padding2`? or even same padding everywhere?

File: src/Ui/Text.elm [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+customParagraph : TextStyle -> String -> Styled.Html msg
+customParagraph font =
+    styledCustomParagraph [] font
+
+
+styledCustomParagraph : List (Styled.Attribute msg) -> TextStyle -> String -> Styled.Html msg
+styledCustomParagraph attrs font s =
+    Styled.p
+        ((Attributes.css <|
+            Css.lineHeight (Css.num 1.5)
```
Comment (balazs-lengyel) [inline src/Ui/Text.elm:59]: drop or configurable

File: src/Ui/Icon.elm [hunk 4]
```diff
@@ -102,11 +100,11 @@ chevronRight =
         , attributes =
             [ SvgAttr.x "0px"
             , SvgAttr.y "0px"
-            , SvgAttr.viewBox "0 0 7.8 13"
+            , SvgAttr.viewBox "0 0 15 26"
             ]
         , children =
             [ path
-                [ SvgAttr.d "M0.4,0.4c-0.5,0.5-0.5,1.4,0,1.9l4.2,4.2l-4.2,4.2c-0.5,0.5-0.5,1.4,0,1.9s1.4,0.5,1.9,0l5.1-5.1l0,0\n\tC8,6.9,8,6.1,7.5,5.5l0,0l0,0l0,0L2.4,0.4C1.8-0.1,0.9-0.1,0.4,0.4z"
+                [ SvgAttr.d "M1.15076 22.6979L10.9885 12.8602L1.15076 3.022C0.543998 2.4079 0.54986 1.41812 1.16395 0.811357C1.77267 0.209966 2.75219 0.209966 3.3614 0.811357L14.3047 11.7546C14.9153 12.3653 14.9153 13.3551 14.3047 13.9653L3.3614 24.9085C2.74731 25.5153 1.75753 25.5094 1.15076 24.8949C0.549372 24.2861 0.549372 23.3066 1.15076 22.6974V22.6979Z"
```
[the remark this answers is not in the document]
Comment (matbyl) [reply src/Ui/Icon.elm:107]: Not right now. This is correct in this case. I will check with Fredrik and update Figma first and make sure we have a consistent use of chevron icons.

File: src/Ui/Input.elm (hunk 2 above)
Comment (matbyl) [reply src/Ui/Input.elm:20]: padding2 is good

**Pull request merged**